### ISSUE TITLE
hello-world

### ISSUE THREAD

**Issue by Bebysys** [Bebysys/Bebysys1#1]
#  первый репозиторий на GitHub

import numpy as np
import matplotlib.pyplot as plt
from tensorflow import keras
from tensorflow.keras.layers import Dense
c = np.array([-40, -10, 0, 8, 15, 22, 38])
f = np.array([-40, 14, 32, 46, 59, 72, 100])
model = keras.Sequential()
model.add(Dense(units=1, input_shape=(1,), activation='linear'))
model.compile(loss='mean_squared_error', optimizer=keras.optimizers.Adam(0.1))
log = model.fit(c, f, epochs=500, verbose=False)
plt.plot(log.history['loss'])
plt.grid(True)
plt.show()
print(model.predict([100]))
print(model.get_weights())

**Comment by Bebysys** [comment]
My 1 project.![Screenshot_2023-03-27-11-46-38-909_com.google.android.apps.docs.jpg](https://user-images.githubusercontent.com/126241946/229346415-58e46158-a0ee-48e9-86d7-1b3b3648467b.jpg)
